src/views/v2/components/ical-link.php

```diff
@@ -24,5 +24,8 @@
 		class="tribe-events-c-ical__link"
 		title="<?php echo esc_attr( $ical->link->title ); ?>"
 		href="<?php echo esc_url( $ical->link->url ); ?>"
-	><?php echo esc_html( $ical->link->text ); ?></a>
+	>
+		<?php $this->template('components/icons/plus', [ 'classes' => ['tribe-events-c-ical__link-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: 03d7a77c491b01416d04e17509c6d17a</sub>